forch/forchestrator.py

```diff
@@ -220,10 +221,27 @@ def _initialize_orchestration(self):
             self._initialize_faucetizer(sequester_segment)
             self._faucetizer.reload_structural_config()
 
+            tail_acl = self._config.orchestration.tail_acl
+            if tail_acl and not self._faucetizer.validate_tail_acl_config(tail_acl):
+                error_msg = 'DVA auth was disabled due to missing ACL for tail_acl config'
```

Comment:
We need to be consistent here.  Is "DVA" an actual feature of Forch or is it a collection of features?  If it's a collection, then this error message should be explicit about what is actually disabled.

Reply:
Update the message to "All auth was disabled...".